src/style.css

```diff
@@ -0,0 +1,112 @@
+html {
+  --main-color: #00acdc;
+  --text-accent-color: #060b35;
+  --text-secondary-color: #616070;
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--text-secondary-color);
+}
+
+body {
+  margin: 0;
+}
+
+.card {
```

Comment:
proofread your code with checklist and task requerements
<img width="508" alt="image" src="https://user-images.githubusercontent.com/102411130/234206351-5de175ee-de39-424f-84d7-29e41cb96c8e.png">